src/transforms/log_to_metric.rs

```diff
@@ -35,6 +48,8 @@ const ORIGIN_SERVICE_VALUE: u32 = 3;
 pub struct LogToMetricConfig {
     /// A list of metrics to generate.
     pub metrics: Vec<MetricConfig>,
+    /// this flag will process all metrics to logs, if defined the `metrics` field is ignored.
```

Comment:
For testing how the generated docs will look like:
```
make generate-component-docs
```

Then to test the changes locally:
```
cd website && make run-production-site-locally
```

And finally navigate to:
http://127.0.0.1:1313/docs/reference/configuration/transforms/log_to_metric/

Reply:
Updated and pushed.